packages/@aws-cdk/core/test/test.stack.ts

```diff
@@ -14,11 +14,11 @@ export = {
     const stack = new Stack();
     stack.templateOptions.templateFormatVersion = 'MyTemplateVersion';
     stack.templateOptions.description = 'This is my description';
-    stack.templateOptions.transform = 'SAMy';
+    stack.templateOptions.transforms = ['SAMy'];
     test.deepEqual(toCloudFormation(stack), {
       Description: 'This is my description',
       AWSTemplateFormatVersion: 'MyTemplateVersion',
-      Transform: 'SAMy'
+      Transform: ['SAMy']
```

Comment:
Just for the sake of not incurring a diff on legacy users, maybe we should render a string instead of an array if `transforms` includes a single item?